backend/Epicenter.Application/DTO/Requests/RecognitionRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Epicenter.Application.DTO.Requests
```

Comment:
I think we should wrap all dto's in some kind of folder, either Infrastructure or Models

Reply:
Same, wasn't sure about this either
But fuck knows what to call it.

Reply:
Done